packages/jest-config/src/readConfigFileAndSetRootDir.ts

```diff
@@ -116,6 +116,7 @@ async function registerTsNode(): Promise<Service> {
     return tsNode.register({
       compilerOptions: {
         module: 'CommonJS',
+        moduleResolution: 'Node',
```

Comment:
Would recommend going with `Node10`, which is the "new" name for this value.

Reply:
Good idea!

Reply:
Addressed in 2a7a0cf5f46a0d343751336197b8f61b4a99dc75